cli/azd/pkg/input/asker.go

```diff
@@ -127,36 +131,34 @@ func askOnePrompt(p survey.Prompt, response interface{}, isTerminal bool, stdout
 		*pResponse = result
 		return nil
 	case *survey.Select:
-		for {
```

Comment:
Does removing this loop impact the desired behavior for real world usage?

Reply:
It impacts real-world usage, but I believe it is the desired behavior.

The example is reflected below:

To run `azd init`, the following stdin prompts needs to be provided:

`Minimal\n<azd env name>\n`

If the user had instead supplied something incorrect like:

`minimal template\n<azd env name>\n`

`azd` would have done:
```
Select a template? error: minimal template is not an allowed choice
Select a template? error: <azd env name> is not an allowed choice
Select a template? error:  is not an allowed choice
Select a template? error:  is not an allowed choice
Select a template? error:  is not an allowed choice
Select a template? error:  is not an allowed choice
...
```

Stalling forever in an infinite loop.